WordPress/wordpress-playground
# Local Environment: Use separate .wp-now directory for test context

Fixes https://github.com/WordPress/wordpress-playground/issues/362



## What?


In this PR, I propose to use a separate wp-now root directory for the test execution context.

## Why?



We don't want to store test artifacts in the `~/.wp-now` directory.

## How?


I introduce a custom function that returns wp-now home directory. It returns a directory placed in the system /tmp directory for test context.

Additionally, I extract two other directory functions from the constants file to a) avoid adding circular dependency there and b) to keep only real constants there.

## Testing Instructions






1. Run tests
```
nx test wp-now --skip-nx-cache
```
2. Confirm that new content was not added in the wp-content directory in main wp-now root directory
```
ls -al ~/.wp-now/wp-content
```
